src/docs/issue-platform-detectors.mdx

```diff
@@ -0,0 +1,137 @@
+---
+title: "Issue Platform - Writing Detectors"
+---
+
+# Fingerprinting
+
+Error events are grouped based on a set of strategies like stack traces, etc. These strategies can be customized by users. Error fingerprinting is a complex ever-evolving mechanism that has been around for many years.
+
+For new kinds of issues, starting with a simpler one-level grouping strategy is advised. Fingerprinting strategies can and should evolve, so an incremental approach here works the best. It’s hard to anticipate all edge cases until you can start fingerprinting real data.
+
+<aside>
+    📌 **A fingerprint represents a unique problem and its each occurrence has the same root cause and the same fix.** For N+1 Db Query Issues, a fingerprint is a hash of parameterized repeated db span description, source span description, parent span description and a problem class (N+1 db query). <Link to="https://github.com/getsentry/sentry/blob/master/src/sentry/utils/performance_issues/performance_detection.py#L1280">Code</Link>.
+</aside>
+
+If an Issue has 3 events that means that this particular problem has 3 instances and the same cause. Fixing this one issue should ensure that all of those 3 instances won’t occur again.
+
+## Risk: Loose grouping strategy
+
+Let’s say we group all N+1 db query problems so that a fingerprint would be solely based on a problem class. This violates same problem = same fix principle. N+1 db queries for different queries would have different fixes.
+
+## Risk: Strict grouping strategy
+
+Let’s say we group all N+1 db queries that have exactly the same span db description (we won’t parameterize them either). That way if a query contains any dynamic fields like dates, then too many issues will be created. Although those issues have the same fix, they won’t be grouped together. This also violates the fingerprinting principles.
```

Comment:
```suggestion
Suppose we group all N+1 db queries that have exactly the same span DB description (without parameterization). In this case, if a query contains dynamic fields like dates, it would result in too many separate issues. Although these issues have the same fix, they would not be grouped together. This also goes against the principles of fingerprinting.
```